source/gameengine/Ketsji/KX_KetsjiEngine.cpp

```diff
@@ -1054,6 +1054,7 @@ void KX_KetsjiEngine::StopEngine()
 
 		while (m_scenes->GetCount() > 0) {
 			KX_Scene *scene = m_scenes->GetFront();
+			scene->RunExitCallbacks();
```

Comment:
You can call the same function inside `RemoveScheduledScenes` when a scene is removed separately.

Reply:
I think that per scene is more interesting, but i would like to hear opinion from @lordloki, @youle31  and other.

Reply:
Is it silly to trigger the callbacks inside `BL_Converter::RemoveScene` ?

Reply:
We are going to duplicate code by calling RunExitCallbacks and it's why you want to move it in common function BL_Converter::RemoveScene, but this class is not really the best place.
I would prefer use a function like `KX_KetsjiEngine::DestructScene`doing the remove from the converter and the callback but not the remove from m_scenes.